src/prefixmaps/ingest/ingest_bioregistry.py

```diff
@@ -54,24 +46,29 @@ def from_bioregistry(upper=False, canonical_idorg=True, filter_dubious=True) ->
                     strict namespace regular expression
     :return:
     """
-    from bioregistry import get_prefix_map
-
-    ctxt = Context("bioregistry", upper=upper)
-    # We always set use_preferred=True, which ensures that OBO prefixes
-    # are either capitalized (e.g. GO) or use the preferred form (e.g. FBbt)
-    prefix_map = get_prefix_map(priority=priority, use_preferred=True)
-    pm_non_preferred = get_prefix_map(priority=priority, use_preferred=False)
-    pm_miriam = get_prefix_map(priority=["miriam"])
-    for prefix, uri_prefix in prefix_map.items():
-        preferred = prefix not in pm_non_preferred or prefix not in pm_miriam
-        if canonical_idorg:
-            uri_prefix = re.sub(
-                r"^https://identifiers.org/(\S+):$",
-                r"http://identifiers.org/\1/",
-                uri_prefix,
-            )
-        if filter_dubious and not NAMESPACE_RE.match(uri_prefix):
-            logging.debug(f"Skipping dubious ns {prefix} => {uri_prefix}")
+    context = Context("bioregistry", upper=upper)
+    prefix_priority = [
+        #  "obofoundry.preferred",
+        "preferred",
+        # "obofoundry",
+        "default",
+    ]
+    priority = [
+        "obofoundry",
+        "miriam.legacy" if canonical_idorg else "miriam",
```

Comment:
do we need semweb here?

Reply:
bioregistry contains these right?  Is the full list below in the registry?  (I use this context file in Biolink, if I want to replace its import with the "merged" file from this repo, I need to make sure we still have all the prefixes, just coming from a different source).  I know you have some graphics that show the overlap between prefixcommons and bioregistry -- can you point me to that doc or a list of outstanding prefixes in prefixcommons that aren't in the registry if any?)
 https://github.com/prefixcommons/biocontext/blob/master/registry/semweb_context.yaml

Reply:
- List of metaregistries: https://bioregistry.io/metaregistry
- Overlap summary: https://bioregistry.io/summary

The `bioregistry.get_extended_prefix_map` priority keywords take valid metaprefixes from the metaregistries and a small number of special cases like `default`, `miriam.legacy`, and `preferred` that signal some special logic during mapping. 

You might be mixing this up with Bioregistry's collections (https://bioregistry.io/collection/), which just allows for task-specific or topic-specific lists of canonical Bioregistry prefixes to be grouped together. This does not factor into anything here.

Reply:
But if the question was "does the bioregistry contain the prefixes that are in this other semantic web list" then the answer is yes. You can see the semantic web collection to check https://bioregistry.io/collection/0000002. Let me know if something's missing.